transactions/marketV3/mint_and_purchase.cdc

```diff
@@ -17,8 +18,8 @@ transaction(sellerAddress: Address, recipient: Address, tokenID: UInt64, purchas
         destroy minter
 
         let seller = getAccount(sellerAddress)
-        let topshotSaleCollection = seller.getCapability(/public/topshotSaleCollection)
-            .borrow<&{TopShotMarketV2.SalePublic}>()
+        let topshotSaleCollection = seller.getCapability(TopShotMarketV3.marketPublicPath)
+            .borrow<&{Market.SalePublic}>()
```

Comment:
Should this be v3?

Reply:
No, this is correct. The public interface for the two is exactly the same so we wanted to use it instead of creating a new one

Reply:
Got it. Thanks for the explanation!